src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -32,10 +32,10 @@
     [react/view {:style styles/toolbar-chat-view}
      [react/view {:style {:flex-direction :row
                           :flex 1}}
-      (if public?
+      (if (or public? group-chat)
```

Comment:
if a chat is `public?` then it is `group-chat` as well

Reply:
Yes, but not vice versa. So needed to add additional condition here.